packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -466,8 +466,8 @@ class FileCollection extends DocumentCollection {
     }
     if (!childPath) {
       const childDoc = await this.statById(childID)
-      childPath = childDoc.path
-      childDirID = childDoc.dirID
+      childPath = childDoc.data.path
+      childDirID = childDoc.data.dirID
```

Comment:
can you add a test for that?

Reply:
Certainly : https://github.com/cozy/cozy-client/pull/639/commits/70b4d3f9c1d3be3c784bf0cab0eb4e6402190dac